### PR TITLE
Add sample list to container details

### PR DESCRIPTION
**Pull request by romgrk**
https://206.12.92.46/issues/299

Question: container.samples is plural but the container list column is singular. Can a container contain multiple samples?

### CODE REVIEW

**Comment by jleblond** [comment]
Yes a container should container multiple samples.. Then, if we really want to sort/filter on that.. I am not quite sure.. The same questions apply to the column 'children' of containers

**Comment by jleblond** [comment]
I have re-read the issue in Redmine, and the request mentions 'a list identical to the sample search page'. I assume this means having the samples listed as a table... and possibly ordered and filtered as samples?? This should be verified with the team/lab. Initially, Alex requested to have a link that opens the container's samples in a new tab.

**Comment by UlysseFG** [comment]
The plate containers can contain multiple samples yeah... depends on the kind of container. Tube have a single sample at most while plates have multiple samples. 
I like the idea of a link to the container samples which would load the sample page with the container filter set in order to list only samples of this container.

**Comment by romgrk** [comment]
I've talked with Alexandre Belisle and I think this is working as expected. Waiting for confirmation but otherwise this can be merged.

**Comment by romgrk** [comment]
From Alexandre Belisle: would like to be table to open samples in new tab, eg with ctrl+click.

**Comment by jleblond** [comment]
@romgrk @UlysseFG this is what I suggest to address the immediate issue of not being able to right click (whether it is a container containing samples or other containers). Dont hesitate to clean up if you think of a better way, and merge in master when approved.

File: src/components/containers/ContainersDetailContent.js [hunk 1]
```diff
@@ -60,7 +60,25 @@ const ContainersDetailContent = ({containersByID, samplesByID, get, listParents}
...
+            {container.children && container.children.map((childId, i) =>
+              <>
+                <Link key={childId} to={`/containers/${childId}`}>
+                  {withContainer(containersByID, childId, container => container.name, <span>Loading…</span>)}
+                </Link>
+                {i !== container.children.length - 1 &&
+                ', '
+                }
+              </>
+            )
```
Comment (romgrk) [inline src/components/containers/ContainersDetailContent.js:76]: Not crucial but you can use [intersperse](https://selfrefactor.github.io/rambda/#/?id=intersperse), it's a bit more declarative.

File: src/components/containers/ContainersDetailContent.js [hunk 2]
```diff
@@ -60,7 +60,25 @@ const ContainersDetailContent = ({containersByID, samplesByID, get, listParents}
           </Descriptions.Item>
           <Descriptions.Item label="Kind">{container.kind}</Descriptions.Item>
           <Descriptions.Item label="Comment" span={3}>{container.comment}</Descriptions.Item>
-          <Descriptions.Item label="Sample(s)" span={3}>
+          <Descriptions.Item label="Content" span={3}>
+            {(container.children && container.children.length > 0 )?
+              <div><b>Container(s)</b></div>
+              : ""}
```
Comment (romgrk) [inline src/components/containers/ContainersDetailContent.js:66]: Usually the form `{condition && element}` is preferred over `{condition ? element : ""}`.

**Review by romgrk — COMMENTED** [review]
See other comments but otherwise I'd merge it.

**Pull request merged**